infrastructure/cdn-in-a-box/traffic_ops/config.sh

```diff
@@ -122,7 +122,13 @@ cat <<-EOF >/opt/traffic_ops/app/conf/cdn.conf
     "geniso" : {
         "iso_root_path" : "/opt/traffic_ops/app/public"
     },
-    "inactivity_timeout" : 60
+    "inactivity_timeout" : 60,
+    "SMTP" : {
+        "Enabled" : false,
```

Comment:
I've seen that in places, but is it convention? I just assumed that it was done because the JSON parser in Perl doesn't care about the case, and Perl's variable naming convention (or at least the one we used in Perl) was `snake_case`. This just matches the naming conventions of Go, allowing the structures to unmarshal "naturally" without struct tags.

Reply:
from what I have seen yes it is convention for json (also the rest of the configure options in this point to that as well. )

Reply:
+1 on keeping the case consistent in the config file, i.e. using snake_case everywhere.

-1 on unmarshalling in Go without struct tags anyway. Explicit is better than Implicit.

Reply:
I don't feel strongly about it - I've pushed up changes to lower/snake case *most* things, and everything is using explicit struct tags. Is it all right to leave the abbreviation capitalized? I won't object to lowercasing it, but figured I'd ask as it's sort of a special case.

Reply:
I think snake_case usually means all lower, including abbreviations.

The Go JSON unmarshaller actually falls back to a case-insensitive match, so `SMTP` and `"smtp"` actually work. But it doesn't collapse underscores, so `InactiveTimeout` and `"inactive_timeout"` would fail: https://play.golang.org/p/4ZJh1bN81JA

Which is why I prefer always using struct tags - because the behavior isn't always obvious to readers who haven't memorized the spec.

Reply:
fiiiiiine
done